app/models/admin.rb

```diff
@@ -31,18 +31,17 @@ class Admin
   devise :database_authenticatable, :recoverable, :rememberable,
     :trackable, :validatable
 
-  def super?
-    self.super_user == true
-  end
-
   def as_json(options = {})
-    super(options).tap do |json|
-      json[:last_sign_in_at] = last_sign_in_at
-
-      if current_admin = options[:current_admin]
-        json[:current_admin] = current_admin.id == id
-      end
-    end
+    current_admin_id = options[:current_admin].try(:id)
+    {
+      _id: id.to_s,
```

Comment:
Ok fair enough. I probably would choose `id` over `_id` but for now `_id` is fine - stay consistent. Then we can change them all later if we want to.